oximeter/collector/src/bin/oximeter.rs

```diff
@@ -36,12 +55,71 @@ enum Args {
         #[clap(name = "CONFIG_FILE", action)]
         config_file: PathBuf,
 
+        /// The UUID for this instance of the `oximeter` collector.
         #[clap(short, long, action)]
         id: Uuid,
 
+        /// The socket address at which `oximeter`'s HTTP server runs.
         #[clap(short, long, action)]
         address: SocketAddrV6,
     },
+
+    /// Run `oximeter` in standalone mode for development.
+    ///
+    /// In this mode, `oximeter` can be used to test the collection of metrics
+    /// from producers, without requiring all the normal machinery of the
+    /// control plane. The collector is run as usual, but additionally starts an
+    /// API server to stand-in for Nexus. The registrations of the producers and
```

Comment:
to be clear: this is a server mock?

Reply:
That's the word I was looking for! :)

Yes, this is a mock, which only supports the 2 endpoints needed to connect producers and collectors. I'll update the docstring here to make this clear. But would it also be better to rename the type from "standalone" to "mock"?

Reply:
If it's just the 2 endpoints, I think it's probably better to not call it a mock, since maybe that implies more than what it is? Anyway I think this is fine as is!